### ISSUE TITLE
Milestone 4 - Discussion on which license to use in the package.

### ISSUE THREAD

**Issue by morrismanfung** [UBC-MDS/prelim_eda_helper#68]

**Comment by morrismanfung** [comment]
Regarding the license to be used for the current package and the corresponding package for R [link](https://github.com/UBC-MDS/R_Prelim_Eda_Helper), I proposed using the [MIT License](https://choosealicense.com/licenses/mit/). I prefer using it as its simple and without much restrictions. A possible alternative is the [GNU General Public License v3.0](https://choosealicense.com/licenses/gpl-3.0/). It is however more restrictive and I do not see the value of using it in our packages, given that some also considers GPL to be a turnoff.

**Comment by austin-shih** [comment]
I agree with @morrismanfung, using the MIT License is straight forward and simple. Considering the scope and nature of the R and Python package created for this course, it is my opinion that we do not require anything more substantial than what we currently have.

**Comment by MNBhat** [comment]
I am also in favor of the MIT license. Even in addition to being restrictive, for this package GPL appears to be too complex to follow.
